app/test_split_integer.py

```diff
@@ -2,20 +2,36 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(8, 1)) == 8,\
+        "Sum of the parts should be equal to the original value."
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(8, 4) == [2, 2, 2, 2],\
```

Comment:
If you use different data sets on the same test it would make sense to parametrize them to avoid code repetition. 
You can optionally use mark,parametrize if you want to be a :star: (it is not required by task definition)